.github/workflows/cf-deploy.yml

```
@@ -1,10 +1,13 @@
 name: Deploy Worker
 on:
   push:
-    branches:
-      - main
-  pull_request:
-  repository_dispatch:
+  #   branches:
+  #     - main
+  # pull_request:
+  # repository_dispatch:
+  branches:
```

actionlint (1.7.12)

[error] unknown Webhook event "branches". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)